src/msw/uilocale.cpp

```diff
@@ -481,6 +481,26 @@ class wxUILocaleImplName : public wxUILocaleImpl
                                     : LOCALE_SDECIMAL);
                 break;
 
+            case wxLOCALE_MEASURE_METRIC:
+                str = DoGetInfo(LOCALE_IMEASURE);
+                if (str.IsEmpty())
```

Comment:
We prefer using standard-like functions:
```suggestion
                if (str.empty())
```